apps/src/templates/teacherDashboard/shapes.jsx

```diff
@@ -18,6 +18,33 @@ export const sectionShape = PropTypes.shape({
   providerManaged: PropTypes.bool.isRequired,
 });
 
+// Used on the Teacher Dashboard for components that
+// require more specific information.
+export const summarizedSectionShape = PropTypes.shape({
+  assignedTitle: PropTypes.string.isRequired,
+  code: PropTypes.string.isRequired,
+  course_id: PropTypes.number,
```

Comment:
nit: can we convert all of these attributes from snake case (`course_id`) to camel case (`courseId`)? i'm not sure how heavy a lift that is, but i'd like all of this to be consistent on the javascript side

Reply:
yeah, that makes sense. that sounds good to me!